### PR TITLE
Create stub for Christof's hometown file

### PR DESCRIPTION
**Pull request by christof-binder**
Should address #9

### CODE REVIEW

File: christof-basel.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Basel
```
Comment (hibellm) [inline christof-basel.md]: Is this correct or should it be Bâle???

Comment (christof-binder) [reply christof-basel.md]: I'll update it to show both

Comment (schickimiki) [reply christof-basel.md]: So you're a city kid?

**Review by hibellm — APPROVED** [review]

**Review by schickimiki — APPROVED** [review]
I am approving. I had to select the radiobox appropriately so it approves when submitting the review.

**Pull request merged**